src/Pim/Bundle/ReferenceDataBundle/Doctrine/ORM/Repository/ReferenceDataRepository.php

```diff
@@ -90,6 +90,18 @@ public function findOneByIdentifier($identifier)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function findByIdentifiers(array $referenceDataCodes)
+    {
+        return $this->createQueryBuilder($this->getAlias())
```

Comment:
you should test if $referenceDataCodes is not empty. I'm not sure but if I remember well, doctrine will throw an exception if array is empty (it does not like to have `IN ()`)